qiskit_nature/second_q/circuit/library/initial_states/hartree_fock.py

```diff
@@ -28,13 +28,13 @@ class HartreeFock(QuantumCircuit):
 
     def __init__(
         self,
-        num_spin_orbitals: int,
+        num_spatial_orbitals: int,
```

Comment:
We had discussed making this a BlueprintCircuit so it could be updated later like UCC is in regards of orbitals etc. Maybe a separate issue could be raised so as to capture the thought.

Reply:
Tracked by https://github.com/Qiskit/qiskit-nature/issues/881